vue/src/components/NoteFooter.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <div
+    class="note-footer"
+  >
+    <span v-if="dueDate !== null">
```

Comment:
v-if can be shrinked to only hide the date, since the icon seems to be always displayed

Reply:
Solved